build

```diff
@@ -61,9 +61,22 @@ log() {
     echo "===< $msg"
 }
 
+delete_unwanted_files() {
+    if [ -e "${1}" ]; then
+        log "Deleting file: ${1}"
+        rm -f "${1}"
+    else
+        log "Cannot delete file: ${1} -- file not found"
+    fi
+}
+
 make_rel() {
     # shellcheck disable=SC1010
```

Comment:
```suggestion
```

Reply:
shellcheck signals SC1010 for both `./rebar3 as "$PROFILE" do release` and `./rebar3 as "$PROFILE" do tar` - disabling in both places is needed

Reply:
https://github.com/emqx/emqx/actions/runs/3419230186/jobs/5692489889#step:4:21

Reply:
remove the `do` should fix it.